README.md

```diff
@@ -8,7 +8,7 @@ You can obtain BeeJU from Maven Central :
 
 # Overview
 BeeJU provides [JUnit rules](http://junit.org/junit4/javadoc/4.12/org/junit/Rule.html) that can be used to write test code that tests [Hive](https://hive.apache.org/). A JUnit rule is a means to provide resources in a test and automatically tear them down when the life cycle of a test ends.
-This project is currently built with and tested against Hive 1.2.1 but is most likely compatible with older and newer versions of Hive. The available JUnit rules are explained in more detail below.  
+This project is currently built with and tested against Hive 2.1.1 (and Hive 1.2.1) but is most likely compatible with older and newer versions of Hive. The available JUnit rules are explained in more detail below.  
```

Comment:
It's now only built against 2.1.1 right? Previous versions were with Hive 1.2.1. Or are we OK with saying this for now as we're confident it will work with 1.2.1? I'd imagine we're only going to support 2.1.1 going forward as this is our use case.